sw/device/tests/aes/aes_test.c

```diff
@@ -0,0 +1,86 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdlib.h>
```

Comment:
Do not use `stdlib.h`; it is not one of the nine freestanding standard library headers.

Reply:
Done.